cmd/tf_operator/main.go

```diff
@@ -13,17 +13,17 @@ import (
 	"github.com/tensorflow/k8s/pkg/controller"
 	"github.com/tensorflow/k8s/pkg/util"
 	"github.com/tensorflow/k8s/pkg/util/k8sutil"
-	"github.com/tensorflow/k8s/pkg/util/k8sutil/election"
-	"github.com/tensorflow/k8s/pkg/util/k8sutil/election/resourcelock"
+	"k8s.io/client-go/tools/leaderelection/resourcelock"
+	election "k8s.io/client-go/tools/leaderelection"
 	"github.com/tensorflow/k8s/version"
 
 	log "github.com/golang/glog"
 
 	"io/ioutil"
 
 	"github.com/tensorflow/k8s/pkg/spec"
-	"k8s.io/client-go/pkg/api/v1"
 	"k8s.io/client-go/tools/record"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
this PR generated LGTM, small nits should format imported package:
```
import (
	"flag"
	"fmt"
	"os"
	"os/signal"
	"runtime"
	"time"
	"io/ioutil"

	log "github.com/golang/glog"
	"github.com/ghodss/yaml"
	"github.com/tensorflow/k8s/pkg/controller"
	"github.com/tensorflow/k8s/pkg/util"
	"github.com/tensorflow/k8s/pkg/util/k8sutil"
	"github.com/tensorflow/k8s/pkg/spec"
	"github.com/tensorflow/k8s/version"

	election "k8s.io/client-go/tools/leaderelection"
	"k8s.io/client-go/tools/leaderelection/resourcelock"
	"k8s.io/client-go/tools/record"
	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
)
```

Reply:
@zjj2wry Thanks for your review, Done